cli/shell/curl.go

```diff
@@ -22,7 +22,7 @@ type HTTPRequest struct {
 	EncodeURL         bool
 	Headers           map[string]string
 	method            string
-	Payload           []byte
+	Payload           string // string representation of fthe payload, in JSON format
```

Comment:
Changing the payload to string, to simplify client code: they. will provide a JSON string instead of the bytes. From now on, the bytes will be obtained internally here.